internal/util/mqclient/rmq_consumer.go

```diff
@@ -18,14 +18,14 @@ import (
 type RmqConsumer struct {
 	c          rocksmq.Consumer
 	msgChannel chan ConsumerMessage
+	closeCh    chan struct{}
 }
```

Comment:
I suggest change `RmqConsumer` to `rmqConsumer` and provider a helper function like newRmqConsumer, in which initializes all the channels.